rollup.config.js

```diff
@@ -1 +1,13 @@
-module.exports = require('skatejs-build/rollup.config');
+const config = require('skatejs-build/rollup.config');
```

Comment:
It'd be ideal to be put in there, I think. If you want to do that as part of this, I think that would be best.